dotnet/src/Connectors/Connectors.AzureOpenAI/Services/AzureOpenAITextToImageService.cs

```diff
@@ -26,41 +28,115 @@ public class AzureOpenAITextToImageService : ITextToImageService
     /// <summary>
     /// Initializes a new instance of the <see cref="AzureOpenAITextToImageService"/> class.
     /// </summary>
-    /// <param name="modelId">The model to use for image generation.</param>
-    /// <param name="apiKey">OpenAI API key, see https://platform.openai.com/account/api-keys</param>
-    /// <param name="organizationId">OpenAI organization id. This is usually optional unless your account belongs to multiple organizations.</param>
-    /// <param name="endpoint">Non-default endpoint for the OpenAI API.</param>
+    /// <param name="deploymentName">Azure OpenAI deployment name, see https://learn.microsoft.com/azure/cognitive-services/openai/how-to/create-resource</param>
+    /// <param name="endpoint">Azure OpenAI deployment URL, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
+    /// <param name="apiKey">Azure OpenAI API key, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
+    /// <param name="modelId">Azure OpenAI model id, see https://learn.microsoft.com/azure/cognitive-services/openai/how-to/create-resource</param>
     /// <param name="httpClient">Custom <see cref="HttpClient"/> for HTTP requests.</param>
     /// <param name="loggerFactory">The <see cref="ILoggerFactory"/> to use for logging. If null, no logging will be performed.</param>
+    /// <param name="apiVersion">Azure OpenAI service API version, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
     public AzureOpenAITextToImageService(
-        string modelId,
-        string? apiKey = null,
-        string? organizationId = null,
-        Uri? endpoint = null,
+        string deploymentName,
+        string endpoint,
+        string apiKey,
+        string? modelId,
```

Comment:
These are unavoidable breaking changes because the `deploymentName` is needed. Was it a bug that we didn't have this included in the earlier code?

Document these as breaking changes and note that the class is experimental.

Reply:
Those are not breaking changes, even though the PR delta looks so. The delta shows the difference between OpenAITextToImageService and AzureOpenAITextToImageService, which is misleading because the OpenAI one does not use a deployment name. 

These service constructor signatures are the same as the current service has, except for the one that accepts AzureOpenAIClient, which I'll document as a breaking change.